configs/configuration.yaml

```diff
@@ -1,17 +1,27 @@
 config_name: cytopipe_defaults
 
+env_manager: conda
+
 # computation configs
 analysis_configs:
   preprocessing:
     threads: 4
 
+# data configurations
+data_configs:
+  plate_data_format: sqlite
+
 # Analysis configuration file paths
 config_paths:
+  # general configs
```

Comment:
What do general configurations provide in context with the other configurations below? Consider providing a description of this somewhere.

Reply:
yes, I agree. I think being more specific about this file name (`configuration.yaml`) might help to clarify as well.

Reply:
I have been thinking of changing the name as well. I will write an issue pertaining to this because I do not know if changing the name of the `configurational` file will cause a domino effect of pathing errors. 

To provide some context. The `configurational` contains the settings for only the `CytoSnake` CLI. This will include presets used to execute a workflow, number of cores, stating data format, etc. Anything that has to do with analysis, those are found within the workflow configs `wf_configs`. 

I am thinking of changing the name to `cytosnake_configs.yaml`

Reply:
Issue created #38

Reply:
I have added documentation at the top of the workflow. Let me know if this is a suitable method to display the workflow documentation.